packages/web/docs/src/components/company-testimonials.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import Image from 'next/image';
+import * as Tabs from '@radix-ui/react-tabs';
+import { ArrowIcon } from './arrow-icon';
+import { CallToAction } from './call-to-action';
+import { Heading } from './heading';
+import { MeetupLogo } from './logos';
+
+type Testimonial = {
+  company: string;
+  logo: (props: { title: string; height: number }) => React.ReactElement;
+  text: string;
+  person: [name: string, title: string, image: string];
+  data: Array<[numbers: string, description: string]>;
+  href: string;
+};
+
+const testimonials: Testimonial[] = [
+  {
+    company: 'Meetup',
+    logo: MeetupLogo,
+    text: 'Hive offers an impressive suite of tools for managing and monitoring GraphQL schemas. The collaborative features, such as schema sharing and team-based permissions, have streamlined our development process.',
+    person: [
+      'Ryan Baldwin',
+      'Senior Backend Engineering Manager',
+      'https://github.com/user-attachments/assets/107be3fa-1051-49d5-b3d7-8fe374a3bd03',
+    ],
+    data: [
+      ['65M+', 'daily events processed'],
+      ['40%', 'more resource efficient'],
+    ],
+    href: '#TODO',
+  },
+  {
+    company: 'Linktree',
+    logo: MeetupLogo,
+    text: 'Hive offers an impressive suite of tools for managing and monitoring GraphQL schemas. The collaborative features, such as schema sharing and team-based permissions, have streamlined our development process.',
+    person: [
+      'Ryan Baldwin',
+      'Senior Backend Engineering Manager',
+      'https://github.com/user-attachments/assets/107be3fa-1051-49d5-b3d7-8fe374a3bd03',
+    ],
```

Comment:
can we just make this object?

```ts
person: {
  name: string,
  title: string,
  avatarUrl: string
}
```

Reply:
Sure. This will probably go to a JSON file. I'll switch all tuples to objects to make it less error-prone.

Do you mind if I keep the same keys I used in the named tuple types?

```ts
type _ = {
  person: Array<{title: string, description: string}> 
  data: Array<[numbers: string, description: string]>
}
```

Reply:
also for person I don't think we need it to be an array. We just need to show one pic at a time